rsts/deployment/cluster_config/performance.rst

```diff
@@ -13,16 +13,49 @@ The video below contains an overview of the Flyte architecture, what is meant by
 Scaling up FlytePropeller
 ==========================
 `FlytePropeller <https://pkg.go.dev/github.com/flyteorg/flytepropeller>`_ is the core engine of Flyte that executes the workflows for Flyte. It is implemented as a `controller <https://kubernetes.io/docs/concepts/architecture/controller/>`__ in Kubernetes.
```

Comment:
Remove "It is implemented as a `controller <https://kubernetes.io/docs/concepts/architecture/controller/>`__ in Kubernetes." otherwise it's duplicated in the next line.

Reply:
done